### PR TITLE
[breaking] drop Wrangler 1 support in favour of Wrangler 2

### LINKED ISSUE #4885: [adapter-cloudflare-workers] Update to support Wrangler v2.x
### Describe the bug

The Cloudflare Workers adapter does not yet support the now generally available Wrangler v2. Wrangler v2 is a _huge_ improvement to v1. The sooner we can all say goodbye to v1 the better. For me, the Cloudflare adapter not yet supporting v2 is the only blocker.

### Reproduction

n/a

### Logs

_No response_

### System Info

```shell
n/a
```

### Severity

blocking an upgrade

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Rich-Harris**
closes #4885. I don't think there's much of a reason to continue supporting Wrangler 1, given Wrangler 2's overwhelming superiority.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: .changeset/little-pots-rule.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-cloudflare-workers': patch
+---
+
+[breaking] support Wrangler 2
```
Comment (Rich-Harris) [inline .changeset/little-pots-rule.md:5]:
```suggestion
[breaking] support Wrangler 2, drop Wrangler 1
```

**Comment by ghostdevv** [comment]
Wrangler 1 has been EOL for 2 days, might be nice to add something in the readme of the cloudflare adapter(s) about requiring Wrangler 2

**Comment by Rich-Harris** [comment]
good catch, updated the README

**Review by benmccann — APPROVED** [review]

**Comment by mglikesbikes** [comment]
Hell yes. Rich, thanks. 👌♥️

**Pull request merged**